feat(AnalyticalTable): add plugin hook to enable disabling selection of rows (#1156)
SAP/ui5-webcomponents-react: packages/main/src/components/AnalyticalTable/AnalyticalTable.stories.mdx

| `responsiveMinWidth` | `number`                                                                 | Defines how the table should react when its width falls below the `responsiveMinWidth`. <ul><li>If `responsivePopIn` is `true` the content of this column will be moved to the first column.</li><li>If `responsivePopIn` is not set or `false` the column will be hidden.</li></ul>                                                                                                                            |
| `PopInHeader`        | `string OR ComponentType`                                                | Custom pop-in header renderer. If set, the table will call that component for every column that is "popped-in" and pass the table instance as prop.                                                                                                                                                                                                                                                             |

<br />

## Plugin Hooks

> Plugin hooks add features to the `Analytical Table` that are meant for special use-cases and are therefore not available per default.

Reviewer: ```suggestion
Plugin hooks are adding features to the `AnalyticalTable` that are meant for special use-cases and are therefore not enabled by default.
```

To add a plugin hook you first need to `import` the corresponding hook and then add it to the `tableHooks` array.

```js
import { useRowDisableSelection } from '@ui5/webcomponents-react/lib/AnalyticalTableHooks';
```

```jsx

> <AnalyticalTable data={data} columns={columns} tableHooks={useRowDisableSelection} />

Reviewer: ```suggestion
<AnalyticalTable data={data} columns={columns} tableHooks={[useRowDisableSelection('myAccessorToDisableRowSelection')]} />
```
